sql/src/main/java/org/apache/druid/sql/calcite/planner/PlannerFactory.java

```diff
@@ -109,6 +109,7 @@ public DruidPlanner createPlanner(
       final SqlEngine engine,
       final String sql,
       final SqlNode sqlNode,
+      final Map<String, Object> userProvidedContext,
```

Comment:
nit: teriminology seems a bit inconsistent across files, some places is `userProvidedContext` and others `userProvidedQueryContext` (and again another place where it would be useful to indicate what is in queryContext and why userProvidedContext exists)